arm-web/2016-08-01/swagger/WebApps.json

```diff
@@ -12218,6 +12335,49 @@
         }
       }
     },
+    "MigrateMySqlStatus": {
+      "description": "MySQL migration status.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "MigrateMySqlStatus resource specific properties",
+          "properties": {
+            "migrationOperationStatus": {
+              "description": "Status of the migration task.",
+              "enum": [
+                "InProgress",
+                "Failed",
+                "Succeeded",
+                "TimedOut",
+                "Created"
+              ],
+              "type": "string",
+              "readOnly": true,
+              "x-ms-enum": {
+                "name": "OperationStatus",
+                "modelAsString": false
```

Comment:
Setting modelAsString to false can result in breaking changes whenever you modify this enum